examples/tcp-client-ssl-tls/tcp_client_ssl_tls.md

```diff
@@ -1,6 +1,6 @@
 # TCP client - SSL/TLS 
 
-This demonstrates how the Ballerina TCP client can be configured to connect to an SSL/TLS listener through a one-way SSL/TLS connection (i.e., the server is verified by the client). 
+A TCP client configured with SSL/TLS configurations connect or interact with a TCP service over an encrypted TLS connection. Provide the `secureSocket` configurations to the client to initiate a secured TCP connection.
 
```

Comment:
**What does it do?**
The TCP client connects to a given TCP server socket, and then sends and receives byte streams.

**How to use it?** 
A TCP client is created by giving the IP and port number. Once connected, `writeBytes` and `readBytes` synchronous methods are used to send and receive byte streams. Since, they are synchronous methods often used in two different strands.  

**When to use it?**
Use this to interact with TCP servers or implement high level protocols based on TCP.